src/modules/db_operations/operations.py

```diff
@@ -58,24 +56,29 @@ def drop_and_populate_strains(app, db):
   drop_tables(app, db, tables=[Strain.__table__])
   load_strains(db)
 
+
 def drop_and_populate_wormbase_genes(app, db, wb_ver: str):
   logger.info(f"Running Drop and Populate wormbase genes with version: {wb_ver}")
-  gene_gff_fname = download_external_db('GENE_GFF_URL', wb_ver=wb_ver)
-  gene_gtf_fname = download_external_db('GENE_GTF_URL', wb_ver=wb_ver)
-  gene_ids_fname = download_external_db('GENE_IDS_URL', wb_ver=wb_ver)
-  homologene_fname = download_external_db('HOMOLOGENE_URL')
-  ortholog_fname = download_external_db('ORTHOLOG_URL')
-  
+
+  file_manager = DatasetManager(wb_ver=wb_ver)
+  gene_gff_fname   = file_manager.fetch_external_db('GENE_GFF_URL', 'c_elegans')
```

Comment:
it would be terrific to get "upgrade" all these hardcoded strings (eg: `GENE_TTF_URL`) with an `enum`.

Reply:
Hardcoded strings are no longer exposed outside the `DatasetManager` class -- now, we can use functions like `DatasetManager.fetch_gene_ttf_db()`.

See: 036af87a48ed2914e6abd3e6510e9a342aad7dc3